libs/openFrameworks/3d/ofNode.cpp

```diff
@@ -231,6 +231,19 @@ void ofNode::rotateAround(float degrees, const ofVec3f& axis, const ofVec3f& poi
 }
 
 //----------------------------------------
+void ofNode::lookAt(const ofVec3f& lookAtPosition){
+    auto relPosition = (getGlobalPosition() - lookAtPosition);
+    auto radius = relPosition.length();
+    if(radius>0){
+        auto latitude = ofRadToDeg(acos(relPosition.y / radius)) - 90;
+        auto longitude = ofRadToDeg(atan2(relPosition.x , relPosition.z));
```

Comment:
it seems to behave properly even then, in the original issue the degenerate case would have x and z as 0 but it still looks ok visually and the rotation values seem correct. atan2 is returning 0 in that case.